backend/migrations/20240214192718_food_nutrition_data.up.sql

```diff
@@ -0,0 +1,16 @@
+-- Add up migration script here
+
+CREATE DOMAIN weight_domain as integer CHECK (VALUE > 0);
+CREATE DOMAIN energy_domain as integer CHECK (VALUE > 0);
```

Comment:
Should they not also be allowed to be zero?

Reply:
do we redundantly call them  `_domain`? we did not do that for the other domains